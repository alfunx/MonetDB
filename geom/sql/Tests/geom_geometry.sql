
CREATE TABLE geoms (g GEOMETRY);
INSERT INTO geoms values ('POINT(10 10)');
INSERT INTO geoms values ('LINESTRING(10 10, 20 20, 30 40)');
INSERT INTO geoms values ('POLYGON((10 10, 10 20, 20 20, 20 15, 10 10))');
INSERT INTO geoms values ('POLYGON((10 10, 10 20, 20 20, 20 15, 10 10), (15 15, 15 20, 10 15, 15 15))');
INSERT INTO geoms values ('LINESTRING(10 10)');
INSERT INTO geoms values ('LINESTRING(10)');
INSERT INTO geoms values ('LINESTRING()');
INSERT INTO geoms values ('LINESTRING');
INSERT INTO geoms values ('');
INSERT INTO geoms values ('POINT(10)');
INSERT INTO geoms values ('POINT()');
INSERT INTO geoms values ('POINT');
INSERT INTO geoms values ('');
INSERT INTO geoms values (NULL);
SELECT * FROM geoms;
DROP TABLE geoms;
