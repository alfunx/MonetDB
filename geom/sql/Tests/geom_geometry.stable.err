stderr of test 'geom_geometry` in directory 'geom/sql` itself:


# 21:15:13 >  
# 21:15:13 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/scratch/monetdb/Linux-x86_64/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB4:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB4/lib:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB4/bin" --set "gdk_dbfarm=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB4/dbfarm" --set "sql_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB4/sql_logs" --set "xquery_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB4/xquery_logs" --set mapi_port=36438 --set xrpc_port=48112 --set monet_prompt= --trace "--dbname=mTests_src_sql" --dbinit="module(geom);module(sql_server); module(sql_server);" ; echo ; echo Over..
# 21:15:13 >  


# 21:15:13 >  
# 21:15:13 >  Mtimeout -timeout 60 MapiClient -lsql -umonetdb -Pmonetdb --host=localhost --port=36438 
# 21:15:13 >  

MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('LINESTRING(10 10)');
ERROR = !IllegalArgumentException: point array must contain 0 or >1 elements
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('LINESTRING(10)');
ERROR = !ParseException: Expected number but encountered ')'
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('LINESTRING()');
ERROR = !ParseException: Expected number but encountered ')'
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('LINESTRING');
ERROR = !ParseException: Expected word but encountered end of stream
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('');
ERROR = !ParseException: Expected word but encountered end of stream
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('POINT(10)');
ERROR = !ParseException: Expected number but encountered ')'
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('POINT()');
ERROR = !ParseException: Expected number but encountered ')'
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('POINT');
ERROR = !ParseException: Expected word but encountered end of stream
MAPI  = (monetdb) /var/tmp/mtest-7831/.s.monetdb.38226
QUERY = INSERT INTO geoms values ('');
ERROR = !ParseException: Expected word but encountered end of stream

# 21:57:03 >  
# 21:57:03 >  Done.
# 21:57:03 >  

