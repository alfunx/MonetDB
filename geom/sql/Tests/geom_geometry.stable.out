stdout of test 'geom_geometry` in directory 'geom/sql` itself:


# 20:49:34 >  
# 20:49:34 >   mserver5 "--config=/home/sjoerd/Monet/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/home/sjoerd/src/MonetDB/devel/geom/x86_64-unknown-linux-gnu/src/monetdb5/.libs:/home/sjoerd/src/MonetDB/devel/geom/x86_64-unknown-linux-gnu/src/monetdb5:/home/sjoerd/Monet/lib64/MonetDB5:/home/sjoerd/Monet/lib64/MonetDB5/lib:/home/sjoerd/Monet/lib64/MonetDB5/bin:/home/sjoerd/Monet/lib64/MonetDB5:/home/sjoerd/Monet/lib64/MonetDB5/lib:/home/sjoerd/Monet/lib64/MonetDB5/bin" --set "gdk_dbfarm=/home/sjoerd/src/MonetDB/devel/geom/x86_64-unknown-linux-gnu/dbfarm" --set "sql_logdir=/home/sjoerd/src/MonetDB/devel/geom/x86_64-unknown-linux-gnu/sql_logs" --set "xquery_logdir=/home/sjoerd/src/MonetDB/devel/geom/x86_64-unknown-linux-gnu/xquery_logs" --set mapi_open=true --set xrpc_open=true --set mapi_port=38689 --set xrpc_port=41370 --set monet_prompt= --trace  "--dbname=mTests_src_sql" --set mal_listing=0 "--dbinit=include geom; include sql;" ; echo ; echo Over..
# 20:49:34 >  

# MonetDB server v5.7.0, based on kernel v1.25.0
# Serving database 'mTests_src_sql', using 2 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://koolmees.mullender.nl:38689/
# MonetDB/GIS module v0.5.0 loaded
# MonetDB/SQL module v2.25.0 loaded

Ready.
#function user.main():void;
#    clients.quit();
#end main;


# 21:57:03 >  
# 21:57:03 >  Mtimeout -timeout 60 MapiClient -lsql -umonetdb -Pmonetdb --host=localhost --port=32181 
# 21:57:03 >  

[ 1	]
[ 1	]
[ 1	]
[ 1	]
[ 1	]
% sys.geoms # table_name
% g # name
% geometry # type
% 0 # length
[ "POINT (10 10)"	]
[ "LINESTRING (10 10, 20 20, 30 40)"	]
[ "POLYGON ((10 10, 10 20, 20 20, 20 15, 10 10))"	]
[ "POLYGON ((10 10, 10 20, 20 20, 20 15, 10 10), (15 15, 15 20, 10 15, 15 15))"	]
[ NULL	]

# 21:57:03 >  
# 21:57:03 >  Done.
# 21:57:03 >  

